clients/java/cac-client/src/main/java/CAC/Client.java

```diff
@@ -0,0 +1,109 @@
+package CAC;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.concurrent.CountDownLatch;
+
+import jnr.ffi.Pointer;
+
+public class Client {
+
+    private static void callCacClient() {
+        String dylib = "cac_client";
+        File currentDir = new File(System.getProperty("user.dir"));
+        String libraryPath = currentDir.getParentFile().getParentFile().getParentFile() + "/target/debug";
+        String tenant = "dev";
+
+        System.out.println("------------------------------------------");
+
+        System.out.println("CAC Client");
+
+        System.out.println("---------------------");
+
+        CacClient wrapper = new CacClient(libraryPath, dylib);
+
+        int newClient;
+        try {
+            newClient = wrapper.cacNewClient(tenant, 1, "http://localhost:8080");
```

Comment:
The URL should not be hardcoded, is this an example file?

Reply:
Yes this is an example file.